lib/vra/catalog.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Author:: Chef Partner Engineering (<partnereng@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'json'
```

Comment:
Usually when you're writing knife plugins you'll want to use Chef::JSONCompat instead of JSON because it uses ffi-yajl.  As this is a  separate gem I think think what you're doing here is fine because depending on chef would be bananas, though you might want to consider using ffi-yajl for compatibility?  Or maybe not, just a thought.